SConstruct

```diff
@@ -645,7 +645,7 @@ if selected_platform in platform_list:
             print(
                 "Detected mingw version is not using posix threads. Only posix "
                 "version of mingw is supported. "
-                'Use "update-alternatives --config <platform>-w64-mingw32-[gcc|g++]" '
+                'Use "update-alternatives --config x86_64-w64-mingw32-[gcc|g++]" '
```

Comment:
"platform" wasn't clear, "arch" would be better but most people will be using this on x86_64 Ubuntu and so giving the explicit command is better.